sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,242 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using Microsoft.CSharp.RuntimeBinder;
+using System.Threading;
+using System.Drawing.Design;
+using System.Reflection;
+
+namespace Azure.ResourceManager.Workloads.Tests.Tests
+{
+    public class SviCRUDTests : WorkloadsManagementTestBase
+    {
+        public SviCRUDTests(bool isAsync) : base(isAsync)
+        {
+            Environment.SetEnvironmentVariable("AZURE_CLIENT_ID", "98f50aef-4bc8-4929-bba1-7d6928d5465a");
+            Environment.SetEnvironmentVariable("AZURE_CLIENT_SECRET", "pqh8Q~dfnHiHb0HwNAOeJ~tu_0QCSD5UFp1NtalD");
+            Environment.SetEnvironmentVariable("AZURE_TENANT_ID", "72f988bf-86f1-41af-91ab-2d7cd011db47");
+            Environment.SetEnvironmentVariable("AZURE_SUBSCRIPTION_ID", "49d64d54-e966-4c46-a868-1999802b762c");
+            Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Record");
+            Environment.SetEnvironmentVariable("AZURE_AUTHORITY_HOST", "https://login.microsoftonline.com");
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        /// <summary>
+        /// The Get Ops Extension Status Interval.
+        /// </summary>
+        private const int GetStatusIntervalinMillis = 10000;
+
+        /// <summary>
+        /// The SVI CRUD test that creates and installs
+        /// an SVI.
+        /// </summary>
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVICrudOperations()
+        {
+            string SviName = "F97";
+            // Create Test RG
+            var resourceGroupName = Recording.GenerateAssetName("SdkRg-SVICrud");
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            ResourceGroupResource rg = await CreateResourceGroup(
+                subscription,
+                resourceGroupName,
+                AzureLocation.EastUS);
+
+            SapVirtualInstanceData SviCreatePayload = this.GetSingleServerPayloadToPut(rg.Data.Name, false);
+
+            // Create SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: SviName,
+                    data: SviCreatePayload);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            SapVirtualInstanceData SviInstallPayload = this.GetSingleServerPayloadToPut(rg.Data.Name, true);
+
+            // Install SVI
+            try
+            {
+                ArmOperation<SapVirtualInstanceResource> resource = await rg.GetSapVirtualInstances().CreateOrUpdateAsync(
+                    waitUntil: WaitUntil.Completed,
+                    sapVirtualInstanceName: SviName,
+                    data: SviInstallPayload);
```

Comment:
I do not see a trace in session with this request. has this code been run and recorded with latest changes?

Reply:
Updated with a new session recording which has install traces.